descriptors/ModuleDescriptor-template.json

```diff
@@ -68,7 +68,7 @@
     },
     {
       "id": "feesfines",
-      "version": "14.0"
+      "version": "14.0 15.0"
```

Comment:
Not a comment but a question - syntactically, having two version numbers separated by a space is valid?

Reply:
Yup. It tells Okapi that mod-patron is compatible with all interface versions in the space separated list.